.eslintrc.cjs

```diff
@@ -66,6 +66,8 @@ module.exports = {
         'rulesdir/cactbot-output-strings': 'error',
         'rulesdir/cactbot-timeline-triggers': 'error',
         'rulesdir/cactbot-response-default-severities': 'error',
+        // Raidboss data files always export a trigger set, and explicit types are noisy.
+        '@typescript-eslint/explicit-module-boundary-types': 'off',
```

Comment:
there is a override option to disable this for trigger files only.

Reply:
I think that's what this is, unless I'm misunderstanding you.

Reply:
I mean we can add a new override rules to disable this for trigger files only instead of disable it at all.

Reply:
Sorry, that's what I'm trying to say.  Isn't this in an override rule for trigger files only already?

Reply:
Oh, sorry 🤣 there isn't.